### DISCUSSION TITLE
Feature request, brotli compression for reading parquet files

### DISCUSSION THREAD

**Discussion by aca** [duckdb/duckdb#5313] [General]
Hi,
Not sure why, but brotli greatly reduces size for my data.
Currently it supports only snappy, gzip, zstd. Support for brotli compression would be really great.

**Comment by hannes** [comment]
It has not come up yet and would incur another dependency. so not sure.

**Reply by m-mohr** [reply]
Here's an example dataset:
https://data.source.coop/fiboa/de-nrw/LFK-AKTI_EPSG25832.parquet (we have a couple more as we used brotli as default compression so far). 

The issue came up for example in
- https://github.com/fiboa/cli/issues/29
- https://github.com/Canner/vulcan-sql/issues/237

**Comment by hannes** [comment]
See #12103

**Reply by m-mohr** [reply]
Wonderful, I appreciate your work! Thank you!

**Reply by cholmes** [reply]
Agreed - this is awesome!

**Comment by hannes** [comment]
The PR for this has been merged into the `feature` branch, so this will be in DuckDB 1.1.0
